### PR TITLE
feat: update structure and create new index.md ES translation

### PR DESCRIPTION
**Pull request by krol3**
### Description
 - New translation : content/es/docs/tasks/debug/_index.md
 - the other files are only moving to keep the new structure 
 

### Issue
Related to https://github.com/kubernetes/website/issues/47444

Closes: #

### CODE REVIEW

File: content/es/docs/tasks/debug/_index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+description: Configurar el monitoreo y el logging para solucionar problemas de un clúster o depurar una aplicación en contenedores.
+weight: 40
+reviewers:
+-
+content_type: concept
+no_list: true
+card:
+  name: tasks
+  weight: 999
+  title: Getting help
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:12]: nit: localize this text

File: content/es/docs/tasks/debug/_index.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+La documentación de este sitio ha sido estructurada para brindar respuestas a una amplia gama de preguntas. [Conceptos](/docs/concepts/) explican la arquitectura de Kubernetes
+y cómo funciona cada componente, mientras [Configuración](/docs/setup/) proporciona
+instrucciones prácticas para empezar. [Tareas](/docs/tasks/) muestran cómo
+realizar tareas de uso común, y [Tutoriales](/docs/tutorials/) son recorridos más completos de escenarios de desarrollo reales, específicos de la industria o de extremo a extremo. La sección de [Referencia](/docs/reference/) proporciona
+documentación detallada sobre el [Kubernetes API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+y las interfaces de línea de comandos (CLIs), tal como [`kubectl`](/docs/reference/kubectl/).
+
+## ¡Ayuda! ¡Mi pregunta no está tratada! ¡Necesito ayuda ahora!
+
+### Stack Exchange, Stack Overflow o Falla del Servidor {#stack-exchange}
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:45]:
```suggestion
### Stack Exchange, Stack Overflow o Server Fault {#stack-exchange}
```

File: content/es/docs/tasks/debug/_index.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+or [InfoSec](https://security.stackexchange.com/questions/tagged/kubernetes).
+
+Es posible que otra persona de la comunidad ya haya hecho una pregunta similar o
+pueda ayudar con su problema.
+
+El equipo de Kubernetes también monitoreará
+[publicaciones etiquetadas con Kubernetes](https://stackoverflow.com/questions/tagged/kubernetes).
+Si no hay ninguna pregunta existente que te ayude, **asegúrate de que tu pregunta
+sea [sobre el tema en Stack Overflow](https://stackoverflow.com/help/on-topic),
+[Falla del Servidor](https://serverfault.com/help/on-topic), o el Stack Exchange
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:66]:
```suggestion
[Server Fault](https://serverfault.com/help/on-topic), o el Stack Exchange
```

File: content/es/docs/tasks/debug/_index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+Una vez que esté registrado, explore la creciente lista de canales para diversos temas de
+interés. Por ejemplo, las personas nuevas en Kubernetes también pueden querer unirse al canal
+[`#kubernetes-novice`](https://kubernetes.slack.com/messages/kubernetes-novice). Como otro ejemplo, los desarrolladores deberían unirse al canal
+[`#kubernetes-contributors`](https://kubernetes.slack.com/messages/kubernetes-contributors).
+
+También hay muchos canales en idiomas locales o específicos de cada país. Siéntete libre de unirte
+estos canales para obtener soporte e información localizados:
+
+{{< table caption="Country / language specific Slack channels" >}}
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:87]: nit: localize this caption

File: content/es/docs/tasks/debug/_index.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+Región | Channels
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:88]:
```suggestion
Región | Canal(es)
```

File: content/es/docs/tasks/debug/_index.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+{{< /table >}}
+
+### Foro
+
+Te invitamos a unirte al Foro oficial de Kubernetes: [discuss.kubernetes.io](https://discuss.kubernetes.io).
+
+### Bugs y solicitudes de funcionalidades
+
+Si tiene lo que parece ser un error (bug) o desea realizar una solicitud de funcionalidades,
+por favor utilice el [Sistema de seguimiento de asuntos en el GitHub](https://github.com/kubernetes/kubernetes/issues).
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:114]: (nit)
```suggestion
por favor utilice el [sistema de seguimiento de asuntos en el GitHub](https://github.com/kubernetes/kubernetes/issues).
```

File: content/es/docs/tasks/debug/_index.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
+---
+title: "Monitoreo, Logging y Depuración"
+description: Configurar el monitoreo y el logging para solucionar problemas de un clúster o depurar una aplicación en contenedores.
+weight: 40
+reviewers:
+-
```
Comment (sftim) [inline content/es/docs/tasks/debug/_index.md:6]:
```suggestion
```

**Review by sftim — COMMENTED** [review]
I recommend some tweaks.

File: content/es/docs/tasks/debug/_index.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+<!-- overview -->
+
+A veces las cosas salen mal. Esta guía tiene como objetivo solucionarlas. Tiene
+dos secciones:
+
+* [Depuración de su aplicación](/docs/tasks/debug/debug-application/) - Útil
+para usuarios que están implementando código en Kubernetes y se preguntan por qué no funciona.
+* [Depuración de su clúster](/docs/tasks/debug/debug-cluster/) - Útil
+para administradores de clústeres y personas cuyo clúster de Kubernetes no está satisfecho.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:23]:
```suggestion
para administradores de clústeres y personas cuyo clúster de Kubernetes no funciona correctamente.
```

File: content/es/docs/tasks/debug/_index.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+realizar tareas de uso común, y [Tutoriales](/docs/tutorials/) son recorridos más completos de escenarios de desarrollo reales, específicos de la industria o de extremo a extremo. La sección de [Referencia](/docs/reference/) proporciona
+documentación detallada sobre el [Kubernetes API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+y las interfaces de línea de comandos (CLIs), tal como [`kubectl`](/docs/reference/kubectl/).
+
+## ¡Ayuda! ¡Mi pregunta no está tratada! ¡Necesito ayuda ahora!
+
+### Stack Exchange, Stack Overflow o Falla del Servidor {#stack-exchange}
+
+Si tienes preguntas relacionadas con *desarrollo de software* para su aplicación en contenedores,
+puedes preguntarles a los de [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:48]:
```suggestion
puedes preguntar en [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes).
```

File: content/es/docs/tasks/debug/_index.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+
+Si tiene preguntas sobre Kubernetes relacionadas con *administración de clusters* or *configuración*,
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:50]:
```suggestion
Si tiene preguntas sobre Kubernetes relacionadas con *administración de clústeres* or *configuración*,
```

File: content/es/docs/tasks/debug/_index.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+puedes preguntarles a los de
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:51]:
```suggestion
puedes preguntarles en
```

File: content/es/docs/tasks/debug/_index.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+[Server Fault](https://serverfault.com/questions/tagged/kubernetes).
+
+También hay varios sitios de Stack Exchange que podrían ser el lugar adecuado para hacer preguntas sobre Kubernetes en áreas como
+[DevOps](https://devops.stackexchange.com/questions/tagged/kubernetes), 
+[Ingeniería de Software](https://softwareengineering.stackexchange.com/questions/tagged/kubernetes),
+or [InfoSec](https://security.stackexchange.com/questions/tagged/kubernetes).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:57]:
```suggestion
o [InfoSec](https://security.stackexchange.com/questions/tagged/kubernetes).
```

File: content/es/docs/tasks/debug/_index.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+[Falla del Servidor](https://serverfault.com/help/on-topic), o el Stack Exchange 
+correcto en el que estás preguntando**, y lea las instrucciones sobre 
+[Cómo hacer una nueva pregunta](https://stackoverflow.com/help/how-to-ask),
+Antes de preguntar una nueva!
+
+### Slack
+
+Muchas personas de la comunidad de Kubernetes se reúnen en Kubernetes Slack en el canal `#kubernetes-users`.
+Slack requiere registro; puede [solicitar una invitación](https://slack.kubernetes.io),
+y el registro está abierto a todos. No dudes en participar y hacer todas y cada una de las preguntas.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:75]:
```suggestion
el registro está abierto a todos. No dudes en participar y hacer todas y cada una de las preguntas.
```

File: content/es/docs/tasks/debug/_index.md [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+Una vez registrado, acceder a la [Organización de Kubernetes en Slack](https://kubernetes.slack.com)
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:76]:
```suggestion
Una vez registrado, accede a la [Organización de Kubernetes en Slack](https://kubernetes.slack.com)
```

File: content/es/docs/tasks/debug/_index.md [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+a través de su navegador web o mediante la aplicación dedicada de Slack..
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:77]:
```suggestion
a través de tu navegador web o mediante la aplicación dedicada de Slack..
```

File: content/es/docs/tasks/debug/_index.md [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+
+Una vez que esté registrado, explore la creciente lista de canales para diversos temas de
+interés. Por ejemplo, las personas nuevas en Kubernetes también pueden querer unirse al canal
+[`#kubernetes-novice`](https://kubernetes.slack.com/messages/kubernetes-novice). Como otro ejemplo, los desarrolladores deberían unirse al canal
+[`#kubernetes-contributors`](https://kubernetes.slack.com/messages/kubernetes-contributors).
+
+También hay muchos canales en idiomas locales o específicos de cada país. Siéntete libre de unirte
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:84]:
```suggestion
También hay muchos canales en idiomas locales o específicos de cada país. Siéntete libre de unirte a
```

File: content/es/docs/tasks/debug/_index.md (hunk 4 above)
Comment (electrocucaracha) [reply content/es/docs/tasks/debug/_index.md:87]:
```suggestion
{{< table caption="País / idioma especifico del canal de Slack" >}}
```

File: content/es/docs/tasks/debug/_index.md [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+Italy | [`#it-users`](https://kubernetes.slack.com/messages/it-users), [`#it-events`](https://kubernetes.slack.com/messages/it-events)
+Japan | [`#jp-users`](https://kubernetes.slack.com/messages/jp-users), [`#jp-events`](https://kubernetes.slack.com/messages/jp-events)
+Korea | [`#kr-users`](https://kubernetes.slack.com/messages/kr-users)
+Netherlands | [`#nl-users`](https://kubernetes.slack.com/messages/nl-users)
+Norway | [`#norw-users`](https://kubernetes.slack.com/messages/norw-users)
+Poland | [`#pl-users`](https://kubernetes.slack.com/messages/pl-users)
+Russia | [`#ru-users`](https://kubernetes.slack.com/messages/ru-users)
+Países Hispanos | [`#es-users`](https://kubernetes.slack.com/messages/es-users)
+Sweden | [`#se-users`](https://kubernetes.slack.com/messages/se-users)
+Turkey | [`#tr-users`](https://kubernetes.slack.com/messages/tr-users), [`#tr-events`](https://kubernetes.slack.com/messages/tr-events)
```
Comment (electrocucaracha) [inline content/es/docs/tasks/debug/_index.md:104]:
```suggestion
Finlandia | [`#fi-users`](https://kubernetes.slack.com/messages/fi-users)
Francia | [`#fr-users`](https://kubernetes.slack.com/messages/fr-users), [`#fr-events`](https://kubernetes.slack.com/messages/fr-events)
Alemania | [`#de-users`](https://kubernetes.slack.com/messages/de-users), [`#de-events`](https://kubernetes.slack.com/messages/de-events)
India | [`#in-users`](https://kubernetes.slack.com/messages/in-users), [`#in-events`](https://kubernetes.slack.com/messages/in-events)
Italia | [`#it-users`](https://kubernetes.slack.com/messages/it-users), [`#it-events`](https://kubernetes.slack.com/messages/it-events)
Japon | [`#jp-users`](https://kubernetes.slack.com/messages/jp-users), [`#jp-events`](https://kubernetes.slack.com/messages/jp-events)
Corea | [`#kr-users`](https://kubernetes.slack.com/messages/kr-users)
Holanda | [`#nl-users`](https://kubernetes.slack.com/messages/nl-users)
Noruega | [`#norw-users`](https://kubernetes.slack.com/messages/norw-users)
Polonia | [`#pl-users`](https://kubernetes.slack.com/messages/pl-users)
Rusia | [`#ru-users`](https://kubernetes.slack.com/messages/ru-users)
Países Hispanos | [`#es-users`](https://kubernetes.slack.com/messages/es-users)
Suecia | [`#se-users`](https://kubernetes.slack.com/messages/se-users)
Turquia | [`#tr-users`](https://kubernetes.slack.com/messages/tr-users), [`#tr-events`](https://kubernetes.slack.com/messages/tr-events)
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Comment by krol3** [comment]
Thanks for the review @sftim and @electrocucaracha. I considered all your comments.

File: content/es/docs/tasks/debug/_index.md [hunk 18]
```diff
@@ -0,0 +1,121 @@
...
+para administradores de clústeres y personas cuyo clúster de Kubernetes no funciona correctamente.
+
+También debe comprobar los problemas conocidos del [release](https://github.com/kubernetes/kubernetes/releases)
+usado.
+
+<!-- body -->
+
+## Obteniendo Ayuda
+
+Si ninguna de las guías anteriores resuelve su problema, existen varias formas de obtener ayuda de la comunidad de Kubernetes..
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:31]:
```suggestion
Si ninguna de las guías anteriores resuelve su problema, existen varias formas de obtener ayuda de la comunidad de Kubernetes.
```

File: content/es/docs/tasks/debug/_index.md [hunk 19]
```diff
@@ -0,0 +1,121 @@
...
+  weight: 999
+  title: Obteniendo Ayuda
+---
+
+<!-- overview -->
+
+A veces las cosas salen mal. Esta guía tiene como objetivo solucionarlas. Tiene
+dos secciones:
+
+* [Depuración de su aplicación](/docs/tasks/debug/debug-application/) - Útil
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:19]:
```suggestion
* [Depuración de tu aplicación](/docs/tasks/debug/debug-application/) - Útil
```

File: content/es/docs/tasks/debug/_index.md [hunk 20]
```diff
@@ -0,0 +1,121 @@
...
+para usuarios que están implementando código en Kubernetes y se preguntan por qué no funciona.
+* [Depuración de su clúster](/docs/tasks/debug/debug-cluster/) - Útil
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:21]:
```suggestion
* [Depuración de tu clúster](/docs/tasks/debug/debug-cluster/) - Útil
```

File: content/es/docs/tasks/debug/_index.md [hunk 21]
```diff
@@ -0,0 +1,121 @@
...
+
+<!-- body -->
+
+## Obteniendo Ayuda
+
+Si ninguna de las guías anteriores resuelve su problema, existen varias formas de obtener ayuda de la comunidad de Kubernetes..
+
+### Preguntas
+
+La documentación de este sitio ha sido estructurada para brindar respuestas a una amplia gama de preguntas. [Conceptos](/docs/concepts/) explican la arquitectura de Kubernetes
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:35]:
```suggestion
La documentación de este sitio ha sido estructurada para brindar respuestas a una amplia gama de preguntas. [Conceptos](/es/docs/concepts/) explican la arquitectura de Kubernetes
```

File: content/es/docs/tasks/debug/_index.md [hunk 22]
```diff
@@ -0,0 +1,121 @@
...
+y cómo funciona cada componente, mientras [Configuración](/docs/setup/) proporciona
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:36]:
```suggestion
y cómo funciona cada componente, mientras [Configuración](/es/docs/setup/) proporciona
```

File: content/es/docs/tasks/debug/_index.md [hunk 23]
```diff
@@ -0,0 +1,121 @@
...
+instrucciones prácticas para empezar. [Tareas](/docs/tasks/) muestran cómo
+realizar tareas de uso común, y [Tutoriales](/docs/tutorials/) son recorridos más completos de escenarios de desarrollo reales, específicos de la industria o de extremo a extremo. La sección de [Referencia](/docs/reference/) proporciona
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:38]:
```suggestion
realizar tareas de uso común, y [Tutoriales](/es/docs/tutorials/) son recorridos más completos de escenarios de desarrollo reales, específicos de la industria o de extremo a extremo. La sección de [Referencia](/es/docs/reference/) proporciona
```

File: content/es/docs/tasks/debug/_index.md [hunk 24]
```diff
@@ -0,0 +1,121 @@
...
+documentación detallada sobre el [Kubernetes API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+y las interfaces de línea de comandos (CLIs), tal como [`kubectl`](/docs/reference/kubectl/).
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:40]:
```suggestion
y las interfaces de línea de comandos (CLIs), tal como [`kubectl`](/es/docs/reference/kubectl/).
```

File: content/es/docs/tasks/debug/_index.md [hunk 25]
```diff
@@ -0,0 +1,121 @@
...
+
+## ¡Ayuda! ¡Mi pregunta no está tratada! ¡Necesito ayuda ahora!
+
+### Stack Exchange, Stack Overflow o Server Fault {#stack-exchange}
+
+Si tienes preguntas relacionadas con *desarrollo de software* para su aplicación en contenedores,
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:46]:
```suggestion
Si tienes preguntas relacionadas con *desarrollo de software* para tu aplicación en contenedores,
```

File: content/es/docs/tasks/debug/_index.md [hunk 26]
```diff
@@ -0,0 +1,121 @@
...
+puedes preguntar en [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes).
+
+Si tiene preguntas sobre Kubernetes relacionadas con *administración de clústeres* o *configuración*,
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:49]:
```suggestion
Si tienes preguntas sobre Kubernetes relacionadas con *administración de clústeres* o *configuración*,
```

File: content/es/docs/tasks/debug/_index.md [hunk 27]
```diff
@@ -0,0 +1,121 @@
...
+
+Es posible que otra persona de la comunidad ya haya hecho una pregunta similar o
+pueda ayudar con su problema.
+
+El equipo de Kubernetes también monitoreará
+[publicaciones etiquetadas con Kubernetes](https://stackoverflow.com/questions/tagged/kubernetes).
+Si no hay ninguna pregunta existente que te ayude, **asegúrate de que tu pregunta
+sea [sobre el tema en Stack Overflow](https://stackoverflow.com/help/on-topic),
+[Server Fault](https://serverfault.com/help/on-topic), o el Stack Exchange 
+correcto en el que estás preguntando**, y lea las instrucciones sobre
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:65]:
```suggestion
correcto en el que estás preguntando**, y lee las instrucciones sobre 
```

File: content/es/docs/tasks/debug/_index.md [hunk 28]
```diff
@@ -0,0 +1,121 @@
...
+pueda ayudar con su problema.
+
+El equipo de Kubernetes también monitoreará
+[publicaciones etiquetadas con Kubernetes](https://stackoverflow.com/questions/tagged/kubernetes).
+Si no hay ninguna pregunta existente que te ayude, **asegúrate de que tu pregunta
+sea [sobre el tema en Stack Overflow](https://stackoverflow.com/help/on-topic),
+[Server Fault](https://serverfault.com/help/on-topic), o el Stack Exchange 
+correcto en el que estás preguntando**, y lea las instrucciones sobre 
+[Cómo hacer una nueva pregunta](https://stackoverflow.com/help/how-to-ask),
+Antes de preguntar una nueva!
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:67]:
```suggestion
antes de preguntar una nueva!
```

File: content/es/docs/tasks/debug/_index.md [hunk 29]
```diff
@@ -0,0 +1,121 @@
...
+
+### Slack
+
+Muchas personas de la comunidad de Kubernetes se reúnen en Kubernetes Slack en el canal `#kubernetes-users`.
+Slack requiere registro; puedes [solicitar una invitación](https://slack.kubernetes.io), el registro está abierto a todos. No dudes en participar y hacer todas y cada una de las preguntas.
+Una vez registrado, ingrese a la [Organización de Kubernetes en Slack](https://kubernetes.slack.com)
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:73]:
```suggestion
Una vez registrado, ingresa a la [Organización de Kubernetes en Slack](https://kubernetes.slack.com)
```

File: content/es/docs/tasks/debug/_index.md [hunk 30]
```diff
@@ -0,0 +1,121 @@
...
+Turquia | [`#tr-users`](https://kubernetes.slack.com/messages/tr-users), [`#tr-events`](https://kubernetes.slack.com/messages/tr-events)
+{{< /table >}}
+
+### Foro
+
+Te invitamos a unirte al Foro oficial de Kubernetes: [discuss.kubernetes.io](https://discuss.kubernetes.io).
+
+### Bugs y solicitudes de funcionalidades
+
+Si tiene lo que parece ser un error (bug) o desea realizar una solicitud de funcionalidades,
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:110]:
```suggestion
Si tienes lo que parece ser un error (bug) o desea realizar una solicitud de funcionalidades,
```

File: content/es/docs/tasks/debug/_index.md [hunk 31]
```diff
@@ -0,0 +1,121 @@
...
+por favor utilice el [sistema de seguimiento de asuntos en el GitHub](https://github.com/kubernetes/kubernetes/issues).
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:111]:
```suggestion
por favor utiliza el [sistema de seguimiento de asuntos en el GitHub](https://github.com/kubernetes/kubernetes/issues).
```

File: content/es/docs/tasks/debug/_index.md [hunk 32]
```diff
@@ -0,0 +1,121 @@
...
+
+Antes de presentar un problema, busque problemas existentes para ver si su problema ya está cubierto.
```
Comment (ramrodo) [inline content/es/docs/tasks/debug/_index.md:113]:
```suggestion
Antes de presentar un problema, busca problemas existentes para ver si tu problema ya está cubierto.
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Some minor suggestions and typos.

**Comment by ramrodo** [comment]
Great!
/lgtm

**Pull request merged**